source/software/matlab_parallel_computing.rst

```diff
@@ -3,12 +3,136 @@
+
+KU Leuven clusters
+------------------
+
+Availability
+++++++++++++
+
+The Parallel Computing Toolbox has been enabled for the Matlab 2022a and 2022b modules on Genius. wICE support is under development at the moment. These versions 
+have been installed in the 2021a toolchain. We strongly recommend using Matlab 2022b installation. 
+
+The Matlab license of KU Leuven covers all academic users. If you would like to use Matlab, but you are not part of our Matlab group and would like to use this 
+software, we kindly ask you to contact our helpdesk and request access.
+
+Configuring Matlab for parallel computing
++++++++++++++++++++++++++++++++++++++++++
+
+Using parallel computing together with Matlab does not work out of the box. You will need to follow some (simple) configurations steps, which will be explained here.
+
+First of all, you will need some scripts that will configure Matlab for parallel use. You can download the appropriate script from this 
+`repository <https://github.com/hpcleuven/matlab-remote>`_. Place this folder somewhere on your cluster storage (e.g. ``$VSC_DATA``). Set your working directory to
+the directory where you saved the scripts. Within the folder, you will see a ``mdcs.rc`` file. This is a configuration file that allows you to adapt the cluster
+profile to your needs. There is only one setting that needs to be changed with the approach you are using here: ``LocalJobStorageLocation``. This will define where
+your job output will be directed to. There is no need for quotation marks when defining this path. **Be careful**: when leaving this field blank, this location will 
+default to your ``$VSC_HOME``, which could result in quickly filling up your home directory.
+
+Next, you'll need to load the module, and start it. It is recommended to do this via an interactive job.
+
+::
+
+   $ # Genius
+   $ module use /apps/leuven/skylake/2021a/modules/all
+   $ module load MATLAB/R2022b #MATLAB/R2022a if you would like to load this version
+    
+Once the module is loaded, start Matlab. You can start it in a terminal without the GUI by using the ``-nodisplay`` flag. Then, run the ``configCluster`` script.
+This will set up the parallel configuration.  
+
+::
+
+   $ matlab -nodisplay
+
+::
+
+   >> rehash toolbox;
+   >> configCluster;
+   
+You will get a message as follows if it succeeded: 'Complete.  Default cluster profile set to "<cluster_name> <matlab_version>".'. 
+   
+Before you will be able to submit multi-node jobs to the cluster, you will first need to get a handle on the cluster, and then set the additional properties of the 
+loaded profile.
+
+::
+
+   >> % get a handle on the cluster
+   >> c = parcluster;
+   >> % If you would prefer to use the local resources (i.e. the resources of the node you are currently on), use:
+   >> c = parcluster('local');
+
+If you would have multiple cluster profiles, you can also load the correct cluster profile with:
```

Comment:
What do you mean with "saving the profile"? As soon as you configure a profile for a cluster-module combo, it will be saved with a standard name "<cluster_name> <matlab_version>". The "local" profile is the profile for when you want to use the resources from the node you are on. 

line 74 illustrates how to load a specific profile, but I am actually thinking of removing this altogether. You can use e.g. 'c=parcluster("Genius R2022a")' instead of using c=parcluster (of course when Genius R2022a is loaded), but there is no reason for it. Matlab will automatically detect which profile it needs to use. So using full profile names can only lead to mistakes.

Reply:
So do you mean that you can only have 1 profile for a specific cluster-Matlabversion combination (in addition to what you could call the "local" cluster profile)? If so, it would indeed be good to make this clear.

Reply:
I have changed extended the explanation starting from 'Configuring Matlab for parallel computing"